pages/planetusermanual.md

```diff
@@ -236,3 +236,40 @@ Notes:
 4. **Open Members Detail** - everyone can see the details of a member here.
 5. **Edit Members Detail** - this is an admin only functionality for editing a members account information.
 6. **Delete a Member** - this is an admin only functionality for deleting members.
+
+# Report Overview
+
+## _What is a Report?_
```

Comment:
make sure to make all the formating uniform to all the other headers on the page.
change ```## _What is a Report?_``` to ```### _What is a Report?_```